### ISSUE TITLE
Can't add a picture to a character, sometimes?

### ISSUE THREAD

**Issue by agentmith** [agnaistic/agnai#385]
It looks like might be limitations to the picture size or format you can use for a character. You can attach a picture when creating a new character, but if it doesn't like the picture it fails to create the chat when you select 'New' on the next screen, and the character is gone when you go back to edit it. If you edit an existing character, it looks like it works on the chat screen, but the picture won't appear if you go back. 

If there are limitations, you might want to consider denying the ability to save and/or providing instructions as to what supported formats/sizes are so you don't lose the character you're creating. I've lost quite a bit of time trying to get characters imported, or by losing characters I've created from scratch before I figured this out.

This is a local install, both with and without MongoDB

**Comment by sceuick** [comment]
The only limitations are for guests due to the localStorage limits which areanywhere from 1MB to 10MB depending on the browser.

**Comment by agentmith** [comment]
I tried testing with a few random images of varying sizes in PNG format, and I just can't get them to attach to a new or existing character. One picture is only 521k and the same thing happens. I've linked the photos I'm testing with. There's practically nothing in my instance other than the default Robot and 1 character I've made myself with a brief test chat. I'm attempting to both create a new test user, and trying to attach the photo to existing user and neither work.

![a pile of robots s--1](https://github.com/agnaistic/agnai/assets/9688832/fc8eb53b-097a-4fbb-b5ea-4e978d666e23)
![a pile of robots s-765438735](https://github.com/agnaistic/agnai/assets/9688832/d6170178-b297-414f-a051-59ea9defc711)
